### DISCUSSION TITLE
The best and worst thing about toonsquid?

### DISCUSSION THREAD

**Discussion by Cjjuntti** [keiwando/toonsquid#614] [General]
So, for me, the best thing about toonsquid is the fact that you get a simpler more intuitive version of adobe animate that you can take with you anywhere you go, all for just ten dollars. The only thing that I think sucks about toonsquid is that it’s only on iPad and the developer says that he has no plans for making it available on other devices. This is just my personal opinion. What do you think about toonsquid? And of course, what do you think sucks about it?

**Comment by Gottafindaname** [comment]
The best is that is works SO well for ipad! The worst is that you can’t copy/paste frames from more layers at a time…

**Comment by Lioncub901** [comment]
Love the animation part of ToonSquid but for some reason cutting a video clip ruins the timing of it so I have constantly fix it 🫠.

**Reply by Lioncub901** [reply]
I love symbols. It makes my life easiler. I do wish there were video transition so I add fades between clips easily and also add other type of transitions (I also like to video edit on ToonSquid).
